modules/cli/pages/index.adoc

```diff
@@ -348,15 +348,29 @@ $ skupper service bind hello-world-backend deployment hello-world-backend
 
 This section shows how to expose a service from a namespace where {skupper-name} is not deployed.
 
+{skupper-name} allows you expose services from other namespaces by default. 
+You can expose other resources, for example deployments if you create a site as described in this section. 
+
 .Prerequisites
 
 * A namespace where {skupper-name} is deployed.
-* A service running in a different namespace.
 * A network policy that allows communication between the namespaces
+* cluster-admin permissions if you want to expose resources other than services
+
+
+. Create a site with cluster permissions if you want to expose a Kubernetes resource other than a service:
```

Comment:
This should be `if you want to expose a workload in a namespace other than that which this skupper site is associated with` or similar.

Reply:
Do we still want users to 'skupper expose service <service>.<namespace>' without `--enable-cluster-permissions`? 
That's what I'm try to say here, skip this step if you only need to expose a service. 
I've tried to address this by introducing workloads in introduction and then use it here